ui/components/ui/button/buttons.scss

```diff
@@ -59,23 +51,23 @@
 }
 
 .btn-warning {
-  color: var(--color-secondary-default);
-  border: 1px solid var(--color-secondary-muted);
-  background-color: var(--color-background-default);
+  color: var(--color-warning-inverse);
+  border: 1px solid var(--color-warning-default);
+  background-color: var(--color-warning-default);
 
   &:hover {
-    border-color: var(--color-secondary-default);
+    border-color: var(--color-warning-alternative);
   }
 
   &:active {
-    background: var(--color-secondary-muted);
-    border-color: var(--color-secondary-default);
+    background: var(--color-warning-alternative);
+    border: 1px solid var(--color-warning-alternative);
   }
 
   &--disabled,
   &[disabled] {
     opacity: 1;
-    color: var(--color-secondary-disabled);
+    color: var(--color-warning-disabled);
```

Comment:
Updating warning button to use warning colors. We want to stay away from using the orange that is associated with the metamask fox logo with any negative connotations e.g. warning, error etc